CollectNewspaperKarel.java

```diff
@@ -13,16 +13,22 @@
 public class CollectNewspaperKarel extends SuperKarel {
 
 	// You fill in this part
-	// Added a comment
-	public void run(){
-		
+	private void goToDoor(){
```

Comment:
This method is still doing more than what its name suggests?

It is going to the door... and it is also stepping out of the door? right?

So stepping out of the door is extra functionality for this method. right?